stdlib/runtime/numbers.gr

```diff
@@ -2489,3 +2489,49 @@ export let isBigInt = x => {
   let x = WasmI32.fromGrain(x)
   isBigInt(x)
 }
+
+// Scalbn is based on https://git.musl-libc.org/cgit/musl/tree/src/math/scalbn.c
+/*
+ * ====================================================
+ * Copyright (C) 2004 by Sun Microsystems, Inc. All rights reserved.
+ *
+ * Permission to use, copy, modify, and distribute this
+ * software is freely granted, provided that this notice
+ * is preserved.
+ * ====================================================
+ */
+@unsafe
+export let scalbn = (x, n) => {
```

Comment:
@spotandjake can you add some graindoc to this to explain the function? Eventually we'll want graindoc on the runtime and having to go back through and figure this stuff out will be annoying.